deploy/README.md

```diff
@@ -29,15 +47,21 @@ To enable Sentry instrumentation for the component, do the following:
 
 ## (Re)Deploying Changes
 
-At the moment we maintain a single demo environment, reachable via a fixed URL (https://otel-demo.testa.getsentry.net/). The environment can be redeployed via our [instance of Argo Workflows](https://run.testa.getsentry.net/). It works as follows:
+At the moment we maintain two demo environments:
+* Staging: https://otel-demo-staging.testa.getsentry.net/
+* Production: https://otel-demo.testa.getsentry.net/
+
+The environments can be redeployed via our [instance of Argo Workflows](https://run.testa.getsentry.net/). It works as follows:
 
 1. Go to https://run.testa.getsentry.net/ (internal-only).
 2. Click "+ Submit New Workflow" in the upper left corner.
 3. Select "opentelemetry-demo-deploy" workflow.
```

Comment:
`4.` is missing. Is this where `... magic ...` goes :P ?

Reply:
Aha, good catch!
Applied a different kind of magic now, for a lil razzle dazzle :)